MediaRecorder.html

```diff
@@ -442,6 +443,38 @@
               <em>Return type:</em> <code>boolean</code>
             </div>
           </dd>
+
+          <dt><dfn><code>getSupportedTypes</code></dfn>, static</dt>
+          <dd>
+            When <code>getSupportedTypes()</code> method is invoked, the UA MUST
+            queue a task, using the DOM manipulation task source, that runs the
+            following steps:
+            <ol>
+            <li>Let <var>p</var> be a promise.</li>
+            <li>Run the following steps in parallel:
+             <ol>
+             <li>Let <var>resultList</var> be an empty list.
+             <li>For each <a href="https://tools.ietf.org/html/rfc2046">MIME
```

Comment:
The [valid MIME type](https://html.spec.whatwg.org/multipage/infrastructure.html#valid-mime-type) concept points to  https://tools.ietf.org/html/rfc7231#section-3.1.1.1 and crucially the syntax there has optional whitespace (OWS) and other optional quoting. There's an example given:

> The quoted and unquoted values are equivalent.  For example, the following  examples are all equivalent, but the first is preferred for consistency:
text/html;charset=utf-8
text/html;charset=UTF-8
Text/HTML;Charset="utf-8"
text/html; charset="utf-8"

So, I think this spec (and EME) needs to constrain the string to the point where no variation is possible, for example by saying that it should be the shortest possible valid MIME type, where every character that can be lowercase is lowercase.